src/DssSpell.sol

```diff
@@ -20,211 +20,202 @@ import {Fileable, ChainlogLike} from "dss-exec-lib/DssExecLib.sol";
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
 import "dss-interfaces/dss/IlkRegistryAbstract.sol";
+import "dss-interfaces/dss/VowAbstract.sol";
 import "dss-interfaces/dss/DogAbstract.sol";
 import "dss-interfaces/dss/ClipAbstract.sol";
 import "dss-interfaces/dss/ClipperMomAbstract.sol";
 import "dss-interfaces/dss/EndAbstract.sol";
 import "dss-interfaces/dss/ESMAbstract.sol";
 
+interface LerpFabLike {
+    function newLerp(bytes32, address, bytes32, uint256, uint256, uint256, uint256) external returns (address);
+}
+
 contract DssSpellAction is DssAction {
 
     // Provides a descriptive tag for bot consumption
     // This should be modified weekly to provide a summary of the actions
-    // Hash: seth keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/6f9bc23e6862ef798bf61349fb06e529703171bd/governance/votes/Executive%20vote%20-%20April%2019%2C%202021.md -q -O - 2>/dev/null)"
+    // Hash: seth keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community//governance/votes/Executive%20vote%20-%20April%2023%2C%202021.md -q -O - 2>/dev/null)"
```

Comment:
```suggestion
    // Hash: seth keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/2cdee5694df5ad59929b55d221f99447c0fe421f/governance/votes/Executive%20vote%20-%20April%2023%2C%202021.md -q -O - 2>/dev/null)"
```

Reply:
Thanks Emilio, but that copy misses the `chop`. As soon as it is fixed I'll add the hash.